src/main/java/com/intuit/graphql/orchestrator/stitching/XtextStitcher.java

```diff
@@ -247,8 +247,8 @@ private GraphQLCodeRegistry.Builder getCodeRegistry(UnifiedXtextGraph mergedGrap
 
         builder.dataFetcher(coordinates,
             dataFetcherContext.getServiceType() == ServiceType.REST
-                ? new RestDataFetcher(serviceMetadata)
-                : new ServiceDataFetcher(serviceMetadata)
+                ? new RestDataFetcher(serviceMetadata, dataFetcherContext.getNamespace())
```

Comment:
Why do we need to pass namespace explicitly? metadata should already have namespace